src/wx/gfxviewers.cpp

```diff
@@ -1582,6 +1582,44 @@ class TileViewer : public GfxViewer {
         }
     }
 
+    void SaveGBATile(wxCommandEvent& ev)
+    {
+        (void)ev; // unused params
+        GameArea* panel = wxGetApp().frame->GetPanel();
+        wxString bmp_save_dir = wxGetApp().frame->GetGamePath(gopts.scrshot_dir);
+        // no attempt is made here to translate the dialog type name
+        // it's just a suggested name, anyway
+        wxString def_name = panel->game_name() + wxT('-') + dname;
+        def_name.resize(def_name.size() - 6); // strlen("Viewer")
+
+        if (captureFormat)
+            def_name += wxT(".bmp");
+        else
+            def_name += wxT(".png");
```

Comment:
Only tangentially related to this, because wx has its own libpng dependency, but we need to replace whatever we use libpng for with stb_image. At some point. To remove one direct dependency of our own.

Reply:
Where are we using it?

Reply:
Apparently `utilWritePNGFile` in `src/Util.cpp`, a few things use that.

Reply:
I will move this topic to #622. I will close this here.